cmd/agent/app/integrations.go

```diff
@@ -444,6 +425,10 @@ func freeze(cmd *cobra.Command, args []string) error {
 
 	tufArgs := []string{
 		"freeze",
+		// We replace the PyPI index with our own by default, in order to prevent
+		// accidental installation of Datadog or even third-party packages from
+		// PyPI.
+		"--index-url", tufIndex,
```

Comment:
Is that necessary for `pip freeze` ? I thought it just read the local metadata

Reply:
Good catch, thanks, it's meant to be for `pip search` --- I'll fix right away!